styx-common/src/main/java/com/spotify/styx/model/data/WFIExecutionBuilder.java

```diff
@@ -167,7 +177,7 @@ public Void terminate(WorkflowInstance workflowInstance, Optional<Integer> exitC
     public Void runError(WorkflowInstance workflowInstance, String message) {
       currWorkflowInstance = workflowInstance;
 
-      executionStatusList.add(ExecStatus.create(eventTs, message));
+      executionStatusList.add(ExecStatus.create(eventTs, "FAILED", Optional.of(message)));
```

Comment:
Can message be null here? Maybe you should use `Optional.ofNullable` in that case.